product-input-fields-for-woocommerce.php

```diff
@@ -244,8 +244,10 @@ public static function ts_admin_notices_scripts() {
 		 * Add tracker completed.
 		 */
 		public static function init_tracker_completed() {
-			header( 'Location: ' . sanitize_text_field( wp_unslash( $_SERVER['HTTP_REFERER'] ) ) );
-			exit;
+			if ( isset( $_SERVER['HTTP_REFERER'] ) ) {
```

Comment:
@PratapDungrani We should handle a case where `$_SERVER['HTTP_REFERER']` is not set.